test/Riok.Mapperly.Tests/Mapping/IgnoreMember.cs

```diff
@@ -0,0 +1,82 @@
+using Riok.Mapperly.Diagnostics;
+
+namespace Riok.Mapperly.Tests.Mapping;
+
+[UsesVerify]
+public class IgnoreMemberTest
+{
+    private readonly string _classA = TestSourceBuilder.CSharp(
+        """
+        class A
+        {
+            public int Value { get; set; }
+
+            [MapperIgnoreMember]
+            public int Ignored { get; set; }
+        }
+        """
+    );
+
+    private readonly string _classB = TestSourceBuilder.CSharp(
+        """
+        class B
+        {
+            public int Value { get; set; }
+
+            [MapperIgnoreMember]
+            public int Ignored { get; set; }
+        }
+        """
+    );
+
+    [Fact]
+    public void ClassAttributeIgnoreMember()
+    {
+        var source = TestSourceBuilder.Mapping("A", "B", TestSourceBuilderOptions.Default, _classA, _classB);
+
+        TestHelper
+            .GenerateMapper(source, TestHelperOptions.AllowInfoDiagnostics)
```

Comment:
Are there actually any info diagnostics reported? If so, these should be asserted...

Reply:
This is not yet fixed, is it? Either remove the `TestHelperOptions.AllowInfoDiagnostics` parameter or assert the reported diagnostics...